apps/ewallet/lib/ewallet/web/paginators/start_after_paginator.ex

```diff
@@ -199,6 +199,12 @@ defmodule EWallet.Web.StartAfterPaginator do
           |> Kernel.<>("Given `#{start_after}` cannot be casted to given `#{start_by}`.")
 
         {:error, :invalid_parameter, msg}
+
+      _ ->
+        # This event should not happen.
+        # We will need to handle more if we encounter this error.
+        {:error, :unknown_error, "An unknown error occured on the database."}
```

Comment:
Would rescuing this cause the error not to be logged? Might need to add `Logger.error("An unknown error occurred during pagination. Error: #{inspect(error)}"`